backend/typescript/services/interfaces/adminService.ts

```diff
@@ -8,24 +13,36 @@ interface IAdminService {
    * @throws Error if waiver creation or update fails.
    */
   updateWaiver(waiver: WaiverDTO): Promise<WaiverDTO>;
+
   /**
    * Get the waiver.
    * @returns A WaiverDTO that contains the waiver.
    * @throws Error if waiver retrieval fails.
    */
   getWaiver(): Promise<WaiverDTO>;
+
   /**
    * Creates a new form template or updates it if it already exists.
    * @returns A FormTemplateDTO that contains the waiver.
    * @throws Error if form template retrieval fails.
    */
   updateFormTemplate(form: FormTemplateDTO): Promise<FormTemplateDTO>;
+
   /**
    * Get the form.
    * @returns A FormTemplateDTO that contains the waiver.
    * @throws Error if form template retrieval fails.
    */
   getFormTemplate(): Promise<FormTemplateDTO>;
+
+  /**
+   * Add a question to the form template
+   * @returns a FormQuestionDTO for the new form
+   * @throws Error if adding the form question fails
```

Comment:
should we also define params? I know we use them in the camper service interface, how come we don't use them here?

Reply:
good catch. I forgot about those